libs/blocks/marquee/marquee.js

```diff
@@ -54,6 +60,16 @@ const decorateBlockBg = (block, node) => {
   }
 };
 
+export const getMetadata = (el) => [...el.rows].reduce((obj, row) => {
+  if (row.children) {
+    const key = row.children[0].textContent.trim().toLowerCase();
+    const content = row.children[1];
+    const text = content.textContent.trim().toLowerCase();
+    if (key && content) obj[key] = { content, text };
+  }
+  return obj;
+}, {});
+
```

Comment:
This function shouldn't be added. It's already part of https://github.com/adobecom/milo/blob/main/libs/blocks/section-metadata/section-metadata.js#L59

You should just import it to use it.

Reply:
Removed the function, as the approach to author the focal point changed.